recipes/gulp/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "gulp" %}
+{% set version = "3.9.1" %}
+{% set sha256 = "9616c6bc8ca165375832bdc096260931369d5339edef2e0a4d48bc7a05b04d15" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/gulpjs/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: generic
+  script: 
+    - npm install -g gulp
+ 
+requirements:
+  build:
+    - nodejs
+  run:
+    - nodejs
+       
+test:
+  commands:
```

Comment:
I'd be happier if this had a test that the script actually worked. Something like `npm link gulp && gulp --help` might work.

Reply:
Thanks for the review. I have added this the test/commands